### PR TITLE
A quick fix issue 771

### PR DESCRIPTION
**Pull request by vnnw**
https://github.com/apple/turicreate/issues/771

### CODE REVIEW

File: src/unity/python/turicreate/test/test_graph_analytics.py [hunk 1]
```diff
@@ -146,6 +146,11 @@ def test_pagerank(self):
             self.assertAlmostEqual(m2.pagerank['pagerank'].sum(), 7087.08, delta=1e-2)
             with self.assertRaises(Exception):
                 assert_frame_equal(m.pagerank.topk('pagerank'), m2.pagerank.topk('pagerank'))
+                
+            with self.assertRaises(Exception):
+                pr_out = m2['pagerank']
```
[the remark this answers is not in the document]
Comment (vnnw) [reply src/unity/python/turicreate/test/test_graph_analytics.py]: But it did raise an exception. 
So I made the modification. 
The cause is indeed all the same for this exception.

image

```
print(str(cc))
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-12-ba966a328a22> in <module>()
----> 1 print(str(cc))

~/anaconda3/lib/python3.6/site-packages/turicreate/toolkits/graph_analytics/_model_base.py in __str__(self)
    144
    145     def __str__(self):
--> 146         return self.__repr__()
    147
    148     def _setting_fields(self):

~/anaconda3/lib/python3.6/site-packages/turicreate/toolkits/graph_analytics/_model_base.py in __repr__(self)
    134         descriptions = [(k, _precomputed_field(v)) for k, v in six.iteritems(self._describe_fields())]
    135
--> 136         (sections, section_titles) = self._get_summary_struct()
    137         non_empty_sections = [s for s in sections if len(s) > 0]
    138         non_empty_section_titles = [section_titles[i] for i in range(len(sections)) if len(sections[i]) > 0]

~/anaconda3/lib/python3.6/site-packages/turicreate/toolkits/graph_analytics/_model_base.py in _get_summary_struct(self)
    114
    115         # collect other sections
--> 116         results = [(k, _precomputed_field(v)) for k, v in six.iteritems(self._result_fields())]
    117         methods = [(k, _precomputed_field(v)) for k, v in six.iteritems(self._method_fields())]
    118         settings = [(k, v) for k, v in six.iteritems(self._setting_fields())]

~/anaconda3/lib/python3.6/site-packages/turicreate/toolkits/graph_analytics/connected_components.py in _result_fields(self)
     68     def _result_fields(self):
     69         ret = super(ConnectedComponentsModel, self)._result_fields()
---> 70         ret["number of connected components"] = len(self['component_size'])
     71         ret["component size"] = "SFrame. See m['component_size']"
     72         ret["vertex component id"] = "SFrame. See m['component_id']"

~/anaconda3/lib/python3.6/site-packages/turicreate/toolkits/_model.py in __getitem__(self, key)
    635
    636     def __getitem__(self, key):
--> 637         return self.get(key)
    638
    639     def _get_native_state(self):

~/anaconda3/lib/python3.6/site-packages/turicreate/toolkits/_model.py in __getattribute__(self, attr)
    338             return self._get(attr)
    339         else:
--> 340             return object.__getattribute__(self, attr)
    341
    342 @_six.add_metaclass(RegistrationMetaClass)

AttributeError: 'ConnectedComponentsModel' object has no attribute 'get'
```

[the remark this answers is not in the document]
Comment (TobyRoseman) [reply src/unity/python/turicreate/test/test_graph_analytics.py]: Right, it did raise an error before your change, but it should not raise an error after your change. 

When adding a unit test for a bug fix, you want it to demonstrate the problem and that the problem is now fixed. The unit test should fail before your fix, but pass after your fix.

[the remark this answers is not in the document]
Comment (vnnw) [reply src/unity/python/turicreate/test/test_graph_analytics.py]: @TobyRoseman Thank you for the review. Actually this bug is a very small one, maybe not worthy of a unit test at all. Will you be kind enough to make proper modification as it fits.

File: src/unity/python/turicreate/test/test_graph_analytics.py [hunk 2]
```diff
@@ -146,6 +146,11 @@ def test_pagerank(self):
...
+                assert_frame_equal(m.pagerank.topk('pagerank'), pr_out.topk('pagerank'))
```
[the remark this answers is not in the document]
Comment (znation) [reply src/unity/python/turicreate/test/test_graph_analytics.py]: @TobyRoseman This checks that they are equal, not that they are not. Isn't that useful?

[the remark this answers is not in the document]
Comment (TobyRoseman) [reply src/unity/python/turicreate/test/test_graph_analytics.py]: @znation you're right. I just confused by the name.
@vnnw - ignore my comment here.

File: src/unity/python/turicreate/test/test_graph_analytics.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (vnnw) [reply src/unity/python/turicreate/test/test_graph_analytics.py]: @srikris Not from my side. IMHO, this fix doesn't need a unit test at all.

**Comment by TobyRoseman** [comment]
Closing in favor of #1028

**Pull request closed without merging**